modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -94,6 +94,46 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
     assertEventsContentType();
   });
 
+  it('TINY-8578: getContent text, empty line in div', () => {
+    const editor = hook.editor();
```

Comment:
Maybe this could be a function like `testGetTextContent('input html', 'expected text')` since all of the tests have the same structure. isSafari could be a scoped variable then used.

This would then become:
```
it('TINY-8578: getContent text, empty line', () => testGetTextContent('<div><p></p></div>', ''));
```